### PR TITLE
DBZ-2275: Updated default database history DML filter default to handle additional RDS statements

### PR DESCRIPTION
**Pull request by rhauch**
https://issues.redhat.com/browse/DBZ-2275

Added two more regular expressions to the default database history DML filter. These are intended to catch and filter out `INSERT into mysql.rds_monitor...` statements that RDS may write in the binlog.

Also added a new unit test that checks that the regular expressions do catch some example statements. This unit test loads the value for the `internal.database.history.ddl.filter` from a properties file, since setting the regular expressions via Java string literals requires escaping certain characters that can be error-prone.

### CODE REVIEW

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlTaskContextTest.java [hunk 1]
```diff
@@ -119,6 +120,20 @@ public void shouldCreateTaskFromConfigurationWithWhenNeededSnapshotMode() throws
         assertThat(context.isSnapshotNeverAllowed()).isEqualTo(false);
     }
 
+    @Test
+    public void shouldFilterInternalDmlStatements() throws Exception {
+        // Load from a properties file to eliminate risk of improper escaping in Java literals
+        Properties props = new Properties();
+        props.load(Testing.Files.readResourceAsStream("dml-filter.properties"));
+        config = simpleConfig().with(MySqlConnectorConfig.SNAPSHOT_MODE, SnapshotMode.WHEN_NEEDED)
+                .with("internal.database.history.ddl.filter", props.get("internal.database.history.ddl.filter"))
```
[the remark this answers is not in the document]
Comment (rhauch) [reply debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlTaskContextTest.java]: Good point. I had used this file to verify that a value used in a configuration file had the proper escapes in the regex, but you are correct that this test does not need that.

I've updated the test to use the default, removed the properties file from the PR, and added a few more conditions to the unit test.

**Comment by rhauch** [comment]
@gunnarmorling again it looks like the build failure on CI is unrelated.

**Comment by gunnarmorling** [comment]
Thanks for the update, the change LGTM now.

> @gunnarmorling again it looks like the build failure on CI is unrelated.

Yes, we fight with some intermittent test failures on CI atm., sorry for the hassle. We're sorting them out one by one, but it takes a bit.

The issue in `MySqlTaskContextTest` actually *is*  related, though: we apply automatic formatting to source files and this failed here. Could you please run `mvn process-sources` and add the resulting change, too. Thanks!

**Comment by rhauch** [comment]
Sorry about that. Fixed by adding the changes resulting from `mvn process-sources`.

**Comment by gunnarmorling** [comment]
Squashed and applied. Thanks a lot, @rhauch!

**Pull request merged**